Fails to build on musl.
../glib/tests/cxx.cpp:509:15: error: missing sentinel in function call [-Werror=format=]
g_test_init (&argc, &argv, NULL);

--- a/glib/tests/cxx.cpp
+++ a/glib/tests/cxx.cpp
@@ -505,7 +505,7 @@ test_string_free (void)
 int
 main (int argc, char *argv[])
 {
 #if G_CXX_STD_CHECK_VERSION (11)
-  g_test_init (&argc, &argv, NULL);
+  g_test_init (&argc, &argv, nullptr);
 #else
   g_test_init (&argc, &argv, static_cast<void *>(NULL));
--- a/gio/tests/cxx.cpp
+++ a/gio/tests/cxx.cpp
@@ -59,7 +59,7 @@ int
 main (int argc, char **argv)
 {
 #if G_CXX_STD_CHECK_VERSION (11)
-  g_test_init (&argc, &argv, NULL);
+  g_test_init (&argc, &argv, nullptr);
 #else
   g_test_init (&argc, &argv, static_cast<void *>(NULL));
 #endif
